question-2/index.html

```diff
@@ -2,21 +2,28 @@
 
 <!DOCTYPE html>
 <html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Return first and last name from a form</title>
+    <script>
+      function getFormvalue() {
+        const firstName = document.getElementsByName("fname")[0].value;
```

Comment:
What happens in the case when the element does not exist? Do you think first accessing the element and trying to check if it exists will reduce possibilities of errors?

Reply:
Right. I will fix this